business_objects/embedding.py

```diff
@@ -48,6 +48,10 @@ def get_embedding_id_and_type(project_id: str, embedding_name: str) -> Any:
     )
 
 
+def get_project_embeddings(project_id: str) -> List[Embedding]:
+    return session.query(Embedding).filter(Embedding.project_id == project_id).all()
```

Comment:
might be mistaken but wouldn't get_finished_embeddings be sufficient? it has the project id and a state check. Error embedings or the ones currently being processed aren't necessary fo the update case anyway right?